MAINTAINERS

```diff
@@ -0,0 +1,8 @@
+# containerd/btrfs maintainers
+#
+# As a containerd sub-project, containerd maintainers are also included from https://github.com/containerd/project/blob/main/MAINTAINERS.
+# See https://github.com/containerd/project/blob/main/GOVERNANCE.md for description of maintainer role
+#
+# COMMITTERS
+# GitHub ID, Name, Email address
+"dennwc","Denys Smirnov",""
```

Comment:
@dennwc What's your email address?

Reply:
You could use the public one on my profile: dennwc@pm.me

Reply:
Thanks, listed your email address